XamarinCommunityToolkit/ObjectModel/AsyncCommand.shared.cs

```diff
@@ -106,36 +80,22 @@ public class AsyncCommand : BaseCommand, IAsyncCommand
 			Action<Exception> onException = null,
 			bool continueOnCapturedContext = false,
 			bool allowsMultipleExecutions = true)
-			: base(canExecute, allowsMultipleExecutions)
+			: base(ConvertExecute(execute), canExecute, onException, continueOnCapturedContext, allowsMultipleExecutions)
 		{
-			this.execute = execute ?? throw new ArgumentNullException(nameof(execute), $"{nameof(execute)} cannot be null");
-			this.onException = onException;
-			this.continueOnCapturedContext = continueOnCapturedContext;
 		}
 
 		/// <summary>
 		/// Executes the Command as a Task
 		/// </summary>
 		/// <returns>The executed Task</returns>
-		public async Task ExecuteAsync()
+		public Task ExecuteAsync() => ExecuteAsync(null);
+
+		static Func<object, Task> ConvertExecute(Func<Task> execute)
 		{
-			ExecutionCount++;
+			if (execute is null)
```

Comment:
in the project, we use `==` to check null.
```suggestion
			if (execute == null)
```